src/main/java/org/folio/inventory/resources/InstancesResponse.java

```diff
@@ -30,6 +32,17 @@ public boolean hasRecords () {
     return !success.getResult().records.isEmpty();
   }
 
+  public Map<String, List<RelatedInstance>> getRelatedInstanceMap() {
+    return Collections.unmodifiableMap(relatedInstanceMap);
+  }
+
+  public InstancesResponse setRelatedInstanceMap(
+    Map<String, List<RelatedInstance>> relatedInstanceMap) {
+
+    this.relatedInstanceMap = relatedInstanceMap;
+    return this;
+  }
```

Comment:
I'm glad this is currently a single map. I understand it might be necessary to have the parent / child distinction, like instance relationships, when it comes to matching records.